plugins/inputs/statsd/statsd.go

```diff
@@ -261,6 +263,9 @@ func (s *Statsd) Gather(acc telegraf.Accumulator) error {
 			fields[prefix+"upper"] = stats.Upper()
 			fields[prefix+"lower"] = stats.Lower()
 			fields[prefix+"count"] = stats.Count()
+			if s.FloatTimings {
+				fields[prefix+"count"] = float64(stats.Count())
+			}
```

Comment:
I think this is cleaner and easier to read
```suggestion
			if s.FloatTimings {
				fields[prefix+"count"] = float64(stats.Count())
			} else {
				fields[prefix+"count"] = stats.Count()
			}
```